### PR TITLE
Update tendrl-node-agent requires and add installation details

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-node-agent is the package required for
tendrl-gluster-integration

This patch adds the required details in the spec file
and updates installation instruction

tendrl-bug-id:Tendrl/gluster_integration#87
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @shtripat Please review

File: doc/source/installation.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@ Environment
 
 1. Install Gluster current stable release (https://www.gluster.org/download/)
 2. Install Etcd>=2.3.x && <3.x (https://github.com/coreos/etcd/releases/tag/v2.3.7)
-
+3. Install tendrl-node-agent using the document available at the url
+   https://github.com/Tendrl/node_agent/blob/master/doc/source/installation.rst
+   tendrl-node-agent is the packae which is required on every storage node
```
Comment (r0h4n) [inline doc/source/installation.rst]: typo "packae"

File: tendrl-gluster-integration.spec [hunk 2]
```diff
@@ -21,6 +21,7 @@ Requires: python-gevent
 Requires: python-greenlet
 Requires: pytz
 Requires: tendrl-common
+Requires: tendrl-node-agent
```
Comment (r0h4n) [inline tendrl-gluster-integration.spec]: tendrl-node-agent is always supposed to be present before tendrl-<sds>-integration, please remove this

Comment (TimothyAsirJeyasing) [inline tendrl-gluster-integration.spec]: Done

File: doc/source/installation.rst (hunk 1 above)
Comment (TimothyAsirJeyasing) [inline doc/source/installation.rst]: Sure, i will correct it

File: doc/source/installation.rst [hunk 3]
```diff
@@ -4,7 +4,9 @@ Environment
 
 1. Install Gluster current stable release (https://www.gluster.org/download/)
 2. Install Etcd>=2.3.x && <3.x (https://github.com/coreos/etcd/releases/tag/v2.3.7)
-
+3. Install tendrl-node-agent using the document available at the url
+   https://github.com/Tendrl/node_agent/blob/master/doc/source/installation.rst
+   tendrl-node-agent is the package which is required on every storage node
```
Comment (r0h4n) [inline doc/source/installation.rst]: This is not required, the tendrl-node-agent is assumed to be present and it is the one which triggers installation of tendrl-gluster-integration

**Pull request closed without merging**